src/checkers/inference/model/Serializer.java

```diff
@@ -26,6 +26,11 @@
 
     T serialize(InequalityConstraint constraint);
 
+    T serialize(ComparableConstraint comparableConstraint);
```

Comment:
don't reorder, we can do that in a different PR which also renames S and T for better clarity

Reply:
Revert this.

Reply:
Done